accounts-db/src/accounts_hash.rs

```diff
@@ -619,6 +540,180 @@ impl AccountsHasher<'_> {
         (num_hashes_per_chunk, levels_hashed, three_level)
     }
 
+    // This function is called at the top lover level to compute the merkle. It
```

Comment:
this fn is copied from `fn compute_merkle_root_from_slices<'b, F, T>(`. Do we need the other fn still?

Reply:
Yes, we do. 
compute_merkle_root_from_slices is still used when we compute merkle tree at 2 level and above, where we have all the data in memory already. 

The comments here may be helpful to understand.

```
// This function is called at the top level to compute the merkle hash. It
// takes a closure that returns an owned vec of hash data at the leaf level
// of the merkle tree. The input data for this bottom level are read from a
// file. For non-leaves nodes, where the input data is already in memory, we
// will use `compute_merkle_root_from_slices`, which is a version that takes
// a borrowed slice of hash data instead.
```